app/Db/pu610k.db

```diff
@@ -48,6 +48,28 @@ record(bo, "$(BASE):$(CHAN):ENABLE_RBV") {
     field(ONAM, "Enabled")
 }
 
+record(calc, "$(BASE):CHILLER_OK") {
+    field(DESC, "LPL Glass Rod chillers on")
+    field(INPA, "$(CHILL):1")
+    field(INPB, "$(CHILL):2")
+    field(INPC, "$(CHILL):3")
+    field(PINI, "YES")
+    field(CALC, "A&&B&&C") 
+    field(SCAN, "1 second")
+    field(PROC, "$(BASE):CHILLER_INTERLOCK.PROC")
```

Comment:
I don't understand this line. I think what you want is:

`field(FLNK, "$(BASE):CHILLER_INTERLOCK PP NMS")`

Reply:
yea that PROC is a typo. I still havent found good documentation on the last names after the field declaration. eg. the PP and NMS, what do these do? I have also seen CPP

Reply:
Have you looked over this page before? https://docs.epics-controls.org/en/latest/guides/EPICS_Process_Database_Concepts.html#scanning-specification

Lots of good stuff in there.

Reply:
I guess the sub-page you're looking for is this: https://docs.epics-controls.org/en/latest/guides/EPICS_Process_Database_Concepts.html#channel-access-links

Reply:
Short answer: PP = Process Passive (process this if SCAN is Passive), NMS = No Maximize Severity (don't set the severity to the biggest severity of the input), CPP = Channel access, Process Passive (connect via channel access, and process this record whenever you get a new value).

But yeah, that page. ;-)

Reply:
Haven't seen this page. Looks like good info.

Reply:
I see how the PP would be necessary for an input field in a record. Where you want the input field to filled with data processed by the incoming record. But does the record specified by the FLNK not process if defined in this field and its scan field set to passive? tldr; is the PP in the FLNK superficial and for clarity or is it changing the logic?